dist/src/main/resources/application-rdbmsPostgres.yaml

```diff
@@ -85,6 +85,9 @@ zeebe:
       rdbms:
         # Classname doesn't matter, since this exporter is present via spring bean
         className: RdbmsExporter
+        args:
+          flushInterval: 5000
```

Comment:
Also vor normal databases we should use a lower flushInterval. Otherwise it is as slow as the current impl where Camunda also have a 5s timeout somewhere.